ddl/job_table.go

```diff
@@ -630,15 +631,36 @@ func getJobsBySQL(sess *session, tbl, condition string) ([]*model.Job, error) {
 
 func syncBackfillHistoryJobs(sess *session, uuid string, backfillJob *BackfillJob) error {
 	sql := fmt.Sprintf("update mysql.%s set state = %d where ddl_job_id = %d and ele_id = %d and ele_key = %s and exec_id = '%s' limit 1;",
-		BackfillHistoryTable, model.JobStateSynced, backfillJob.JobID, backfillJob.EleID, wrapKey2String(backfillJob.EleKey), uuid)
+		BackgroundSubtaskHistoryTable, model.JobStateSynced, backfillJob.JobID, backfillJob.EleID, wrapKey2String(backfillJob.EleKey), uuid)
 	_, err := sess.execute(context.Background(), sql, "sync_backfill_history_job")
 	return err
 }
 
-func generateInsertBackfillJobSQL(tableName string, backfillJobs []*BackfillJob) (string, error) {
+func generateInsertBackfillJobSQL(backfillJobs []*BackfillJob) (string, error) {
 	sqlBuilder := strings.Builder{}
 	sqlBuilder.WriteString("insert into mysql.")
-	sqlBuilder.WriteString(tableName)
+	sqlBuilder.WriteString(BackgroundSubtaskTable)
+	sqlBuilder.WriteString("(task_key, ddl_physical_tid, type, exec_id, exec_expired, state, checkpoint, start_time, state_update_time, meta) values")
+	for i, bj := range backfillJobs {
+		mateByte, err := bj.Meta.Encode()
+		if err != nil {
+			return "", errors.Trace(err)
+		}
+
+		if i != 0 {
+			sqlBuilder.WriteString(", ")
+		}
+		sqlBuilder.WriteString(fmt.Sprintf("('%d_%d_%s_%d', %d, %d, '%s', '%s', '%s', %s, %d, %d, %s)",
+			bj.ID, bj.EleID, hex.EncodeToString(bj.EleKey), bj.JobID, bj.StoreID, bj.Tp, bj.InstanceID, bj.InstanceLease, bj.State.String(), wrapKey2String(bj.Meta.CurrKey),
```

Comment:
s/`bj.ID, bj.EleID, hex.EncodeToString(bj.EleKey), bj.JobID`/`bj.JobID,hex.EncodeToString(bj.EleKey), bj.EleID, bj.ID`?
We're going to use a prefix index?